plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -95,10 +95,8 @@ export class GraphQLInstrumentation extends InstrumentationBase {
     return module;
   }
 
-  private _addPatchingExecute(): InstrumentationNodeModuleFile<
-    typeof graphqlTypes
```

Comment:
here the patch function argument is already typed as `any` so the generic type had no effect. trying to type `moduleExports` yields a ts error